nidaqmx_examples/playrec.py

```diff
@@ -0,0 +1,83 @@
+"""Simultaneous read and write with NI USB-4431 or similar device."""
+
+import numpy as np
+import nidaqmx as ni
+from nidaqmx.constants import WAIT_INFINITELY
+
+
+def query_devices():
```

Comment:
This function isn't used.

Reply:
Changed example slightly such that this function is actually useful here.